scripts/paths.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file has been ejected from create-react-app v3.4.0.
+ * The original file was located at `config/paths.js` but has now
+ * been heavily modified and moved here.
+ */
+const fromEntries = require('object.fromentries');
+const utils = require('./utils');
+const resolveApp = utils.resolveApp;
+const resolveModule = utils.resolveModule;
+
+const componentsToExport = [
+    'cell-tooltip',
+    'channels',
+    'factors',
+    'genes',
+    'heatmap',
+    'scatterplot',
+    'sets',
+    'sourcepublisher',
+    'spatial',
+    'status',
+];
+
+module.exports = {
+    appPackageJson: resolveApp('package.json'),
+    appPath: resolveApp('.'),
+    appBuild: resolveApp('build-demo'),
+    appPublic: resolveApp('demo'),
+    appHtml: resolveApp('demo/index.html'),
+    appIndexJs: resolveModule(resolveApp, 'src/demo/index'),
+    appSrc: resolveApp('src'),
+    appJsConfig: resolveApp('jsconfig.json'),
```

Comment:
Does this exist? If not, can we remove?

Reply:
I had never seen it before, it looks like it is a visual studio code thing (https://code.visualstudio.com/docs/languages/jsconfig) so I will remove. @mccalluc  or @manzt is that ok with both of you?

Reply:
yeah, I think the jsconfig.json can go.